src/Stratis.Bitcoin.Features.PoA/Voting/Poll.cs

```diff
@@ -5,12 +5,18 @@
 
 namespace Stratis.Bitcoin.Features.PoA.Voting
 {
+    public class Vote
```

Comment:
It is not better just to make this also `IBitcoinSerializable` ?

Reply:
Done